student.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'person'
+
+class Student < Person
+  def initialize(_id, age, classroom, name = 'Unknown', parent_permission: true)
```

Comment:
Hi @firepicaso, 

Great job on your `Student` class! 🚀 Your code is clean, clear, and showcases a solid understanding of inheritance in Ruby. 👏 Nevertheless, please note that the id parameter is not needed here.

## Points to address: 🔍👨‍🔧

1. **ID Parameter not needed**:
      As per your implementation:
      ```ruby
      def initialize(_id, age, classroom, name = 'Unknown', parent_permission: true)
      ```
      Note that the `id` parameter is not needed here since it will be managed within the `Person` class.

Keep up the brilliant work! 🌟👨‍💻

If you have any questions or need further clarification on the points mentioned, please feel free to ask. Remember to tag me, so I can get back to you ASAP.

Happy coding! 🚀👨‍💻

----------
_Remember, as per the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project. If you believe that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._